content/en/firewall.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Firewall
```

Comment:
Considering this will not be the only firewall definition out there we should be sure to include multiple references to other firewall definitions.

Reply:
Hey Jason! I have worked on the previous two suggestions, but can you please elaborate on this one

Reply:
I think you can ignore this comment for now, @akshatcoder-hash. We'll likely add metadata for further authoritative references.

Reply:
Hey @CathPag , are there suggestions for any more changes in this?

Reply:
Did you address Jason's other comments, @akshatcoder-hash? My note only referred to the authoritative source.